### PR TITLE
Adding Interface & script service examples (I

### PR DESCRIPTION
**Pull request by sauvainr**
Originally to reproduce https://jira.exosite.com/browse/MUR-10139 but issue seems not related.

Also remove the default_script example as the scope of usage is still to be discussed.

### CODE REVIEW

**Comment by guitarmind** [comment]
@sauvainr Is the new example trying to let defined operation and event calling each other? What kind of situation should we use `Scripts.trigger` instead of `Interface.trigger`? This part is a bit confusing to me...

**Comment by sauvainr** [comment]
Ok I ll add more comments.

File: services/interface/interface.yaml [hunk 1]
```diff
@@ -1,17 +1,37 @@
...
+  description: |
+    An operation usable by subscribers.
+    The script handling this call is done in the interface myOp.lua handler.
+  parameters:
+    type: object
+    description: payload
+    properties:
+      token:
+        type: string
+        x-exosite-restricted: true # -> cannot be provided from scripting
```
Comment (guitarmind) [inline services/interface/interface.yaml]: @sauvainr Where would be the place to configure the `token` parameter of operation, and what if it is `x-exosite-restricted`?

Comment (guitarmind) [reply services/interface/interface.yaml]: So `x-exosite-restricted` mean token is not accessible from Lua and configurable from Murano UI?

Comment (sauvainr) [reply services/interface/interface.yaml]: correct

**Comment by sauvainr** [comment]
@guitarmind I updated the doc, please let me know if better.

**Pull request merged**